jetbrains-toolbox.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec env XDG_CONFIG_HOME="$HOME/.config" XDG_DATA_HOME="$HOME/.local/share" XDG_CACHE_HOME="$HOME/.cache" /app/extra/toolbox/bin/jetbrains-toolbox "$@" 
```

Comment:
why are you changing these?

Reply:
This env is required for Toolbox working properly with flatpak, without this env vars Toolbox will download the IDE at `~/.var/app/com.jetbrains.Toolbox/data` and not in  `$HOME/.local/share/JetBrains/Toolbox/apps`. and the location of config file and cache will located at `~/.var/app/com.jetbrains.Toolbox/ config` and `~/.var/app/com.jetbrains.Toolbox/ cache`.